### ISSUE TITLE
`move-line-up/down` command breaks JSX indentation

### ISSUE THREAD

**Issue by jgebhardt** [gandm/language-babel#150]
## Repro steps
Using the following test file:
```jsx
<div>
  <div>
    <p>
      first line
    </p>
  </div>
  <div>second line</div>
</div>
```

Select the first child of the enclosing `<div />`:
![screen shot 2016-03-03 at 12 49 08 pm](https://cloud.githubusercontent.com/assets/632968/13509128/cb7f5ecc-e13e-11e5-9e23-c6d01ed045b1.png)

Use the `editor:move-line-down` / `editor:move-line-up` (cmd+ctrl+up/down) command to move the selection around. This breaks the indentation by setting each line to the first line's indent level.

![screen shot 2016-03-03 at 12 49 17 pm](https://cloud.githubusercontent.com/assets/632968/13509127/cb7c9b74-e13e-11e5-9f33-f3e35a93ccf6.png)

**Comment by gandm** [comment]
OK Thanks. I'll take a look. In the meantime just cursor into the last row moved and it will re-indent the previous rows.

**Comment by gandm** [comment]
@jgebhardt Looks like I'm indenting it correctly but on return  Atom re-indents at https://github.com/atom/atom/blob/master/src/language-mode.coffee#L235 using the previous line's indent. 

I may have to monkey patch this method which I'd hoped to avoid.

**Comment by gandm** [comment]
Incorrect scopes returned by atom.

**Comment by gandm** [comment]
@jgebhardt Should now be fixed in 2.15.6 release.
